pkg/odo/util/completion/completionhandlers.go

```diff
@@ -75,7 +76,7 @@ var ProjectNameCompletionHandler = func(cmd *cobra.Command, args parsedArgs, con
 	for _, project := range projects {
 		// we found the project name in the list which means
 		// that the project name has been already selected by the user so no need to suggest more
-		if val, ok := args.commands[project.Name]; ok && val {
+		if args.commands[project.Name] {
```

Comment:
Is this guaranteed to work?

If so, can you please apply it to the rest of the handlers as well?

Reply:
This works if the map is of type `map[string]bool`, which it is :)

Reply:
Yeah, just found a relevant SO answer :)

However this change should also be applied to the other functions in this file as well